src/Driver/driver_configs.jl

```diff
@@ -252,12 +273,24 @@ function AtmosLESConfiguration(
         meshwarp = meshwarp,
     )
 
+    if cutofforder_horz < polyorder_horz || cutofforder_vert < polyorder_vert
+        filter = CutoffFilter(
+            grid,
+            (cutofforder_horz + 1, cutofforder_horz + 1, cutofforder_vert + 1),
+        )
+    else
+        filter = nothing
```

Comment:
If we fall in this `else` condition, we should change the values of `cutofforder_horz` and `cutofforder_vert` to `nothing`, so that in the print that follows, the user indeed sees that in that case the filter was not applied. Otherwise, the way it is right now, the user would see the attempted cutoff filter degrees, not the actual (nothing) one and it would be misleading.

Reply:
I changed it in [c575dc3](https://github.com/CliMA/ClimateMachine.jl/pull/1953/commits/c575dc3358e3e202eea2e057078fda5b0550c311)

Reply:
Thanks!